tests/conftest.py

```diff
@@ -400,18 +411,26 @@ def _():
     return _
 
 
-@pytest.fixture(name="static_server")
+@pytest.fixture(name="static_server", autouse=True, scope="session")
 def fixture_static_server():
     if os.environ.get("DEBIAN_FRONTEND"):
-        return "http://static-server:8000"
-    return "http://localhost:8000"
+        yield "http://static-server:8000"
```

Comment:
No this doesn't need to be HTTPS. `localhost:8888` needs to be HTTPS because it's the search provider and firefoox wont register it if it's not HTTPS.